client/js/verification-portal/components/time.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import injectSheet from 'react-jss'
+import { connect } from 'react-redux'
+import { change } from 'redux-form'
+
+class Time extends React.Component {
```

Comment:
I don't think there is any reason this needs to be a class. What do you think of using this style?
https://github.com/HelpAssistHer/help-assist-her/blob/master/client/js/components/button.js#L5

Reply:
Is it possible to dispatch changeFieldValue without making it class?